### ISSUE TITLE
Documentation Error - LinkWithLocalMicAndSpeaker

### ISSUE THREAD

**Issue by mmiller7** [sm0svx/svxlink#154]
In the following guide, the location for TCL files is incorrect.
https://github.com/sm0svx/svxlink/wiki/LinkWithLocalMicAndSpeaker

TLDR-version, all the paths in this guide where /usr/share is referenced incorrectly point to svxlik/sound instead of just svxlink directory.

Details:

References to the TCL Logic Core are "/usr/share/svxlink/sounds/events.d/" which does not exist.
I think it should be either "/usr/share/svxlink/events.d" for the source "SimplexLogic.tcl" and either the same or /usr/share/svxlink/events.d/local" for the new "MicSpkrLogic.tcl" file.

Similarly, the EVENT_HANDLER path in the guide for "EVENT_HANDLER=/usr/share/svxlink/sounds/events.tcl" should be fixed to reflect the proper location, which I believe should be "EVENT_HANDLER=/usr/share/svxlink/events.tcl".

EDIT: Additional issue, PTT_TYPE is not specified in your example:
*** ERROR: PTT_TYPE not specified for transmitter TxSpkr. Legal values are: "Dummy" "GPIO" "Hidraw" "PTY" "SerialPin" or "NONE"

I believe the proper value would be PTT_TYPE=NONE

**Comment by sm0svx** [comment]
Thanks for pointing that out. Better now?

**Comment by mmiller7** [comment]
Looks much better now!
